src/Models/FieldMaps/b1_dam.m

```diff
@@ -66,12 +77,21 @@
 
         function obj = b1_dam
             obj.options = button2opts(obj.buttons);
+            obj = UpdateFields(obj);
         end
-
+        
         function FitResult = fit(obj,data)
-            FitResult.B1map = abs(acos(data.SF2alpha./(2*data.SFalpha))./(deg2rad(obj.Prot.Alpha.Mat)));
+            FitResult.B1map_raw = abs(acos(data.SF2alpha./(2*data.SFalpha))./(deg2rad(obj.Prot.Alpha.Mat)));
+            %remove 'spurious' points to reduce edge effects
+            FitResult.Spurious = double(FitResult.B1map_raw<0.5);
```

Comment:
Why are spurious values set to be below 0.5 on this line, but two lines below this one they are those below 0.6?

Reply:
Also, is there a reason why the cutoff is set for only a low number? (i.e. are there no spurious values above something like 1.5-2.0?